src/main/java/org/jenkinsci/plugins/displayurlapi/actions/RunDisplayAction.java

```diff
@@ -11,7 +11,10 @@
 import org.kohsuke.stapler.StaplerRequest;
 
 import java.util.Collection;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
 
+@ExportedBean(defaultVisibility = 0)
```

Comment:
I couldn't figure out how to make it show up by default but with this current config it shows up at depth=1